tests/native_tests/native_macros.hy

```diff
@@ -298,31 +298,6 @@
   (assert (= (lif-not 0 "false" "true") "true")))
 
 
-(defn test-yield-from []
```

Comment:
Weren't these two tests run for both Python 2 and Python 3? Would it make sense to keep them around for Python 3 still?